wiki/Ranking_Criteria/osu!mania/en.md

```diff
@@ -60,6 +60,7 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
 - **There must not be more than 6 notes pressed at any given moment in Insane or lower difficulties.** Using more than 6 notes must also follow a reasonable spread to the next lower difficulty. This rule does not apply to ends of long notes, as they are released, not pressed.
 - **Beatmaps can only use 4 to 10 keys.** Anything else is not supported for the Ranked section.
 - **If multiple key modes are present in a single beatmap, the key mode must be denoted in all difficulty names. Otherwise, the key mode must not be denoted.**
+- **Beatmaps using the special N+1 style of mapping must be denoted as such under the "Advanced" tab of song set-up in the editor and "N+1" must be added to tags. For sets containing multiple keymodes, keymodes of this style must be denoted using their N+1 notation (e.g. using 7K+1 instead of 8K on difficulty names).**
```

Comment:
```suggestion
- **Beatmaps using the special N+1 style of mapping must be denoted as such under the `Advanced` tab of song setup in the editor and "N+1" must be added to the tags. For beatmaps containing multiple key modes, difficulties using this style must be denoted using N+1 notation (e.g. using 7K+1 instead of 8K on difficulty names).**
```